src/pages/campaign/writing-nft.vue

```diff
@@ -124,5 +125,18 @@ export default {
       return LIKECOIN_NFT_CAMPAIGN_ITEMS;
     },
   },
+  mounted() {
+    const { hash } = this.$route;
+    if (hash) {
+      try {
+        const el = document.querySelector(hash);
```

Comment:
What are the expected value of hash? does it need a `#` prefix?

Reply:
The hash is returned with `#` already